lib/dynosaur/process/heroku.rb

```diff
@@ -4,6 +4,13 @@
 module Dynosaur
   class Process
     class Heroku < Process
+      # Heroku private dyno sizes
+      module PrivateSize
```

Comment:
What's the value of a separate module from `Size`?

Reply:
Moved into existing `Size` module.